tests/Mouf/Database/MagicQueryTest.php

```diff
@@ -177,12 +180,13 @@ public function testInvalidSql() {
     public function testWithCache()
     {
         global $db_url;
-        $config = new \Doctrine\DBAL\Configuration();
-        // TODO: put this in conf variable
         $connectionParams = array(
-            'url' => $db_url,
+            'user' => $GLOBALS['db_username'],
+            'password' => $GLOBALS['db_password'],
+            'host' => $GLOBALS['db_host'],
+            'driver' => $GLOBALS['db_driver'],
```

Comment:
You introduce a "db_port" variable in phpunit.xml.dist, but you don't use it here.
Apart for this single change, I'm :+1: for merging this!

Thanks a lot